catalog_publish.java

```diff
@@ -115,7 +116,7 @@ private boolean processCatalog(Path platformYaml) {
                 log.info("Platform is disabled. Skipping");
                 return false;
             }
-            String repository = tree.path("maven-repository").asText(MAVEN_CENTRAL);
+            List<String> repositories = java.util.Arrays.asList(tree.path("maven-repository").asText(MAVEN_CENTRAL).split(","));
```

Comment:
I'm not a fan of this, I'd prefer having the repositories declared in the YAML as lists, which can be tested like in https://github.com/quarkusio/jbang-catalog/blob/main/catalog_publish.java#L228-L242

Reply:
Oh, didn't think of that. I wanted to keep it backwards-compatible but yeah, I guess we can allow the value of the `repository` field to be either a string, or multiple strings, and that should still be backwards-compatible. Ok, I'll re-do it tomorrow